### DISCUSSION TITLE
Welcome to obsidian-confluence Discussions!

### DISCUSSION THREAD

**Discussion by andymac4182** [markdown-confluence/markdown-confluence#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ygaudet1967** [comment]
Hi. I tried to install this using BRAT but get the following error:

BRAT
markdown-confluence/markdown-confluence
The release is not complete and cannot be
download.main.js is missing from the Release

Thanks,
Yo

**Reply by andymac4182** [reply]
Hi,

Sorry I just moved the repositories around and must of missed the docs. The new repo is https://github.com/markdown-confluence/obsidian-integration

**Reply by ygaudet1967** [reply]
Thank you.....that worked.

I'm having issues with configuration. I am in the Obsidian42-BRAT plugin menu settings but I don't see where to put in base URL, parentID, etc.  I see options for autoupdate plugin options, beta plugin list, beta themes list and monitoring. Should I be looking elsewhere?

**Reply by andymac4182** [reply]
Once you download the plugin you can go to community plugins and press refresh. You should see confluence integration then and you can enable it which will give you a settings tab for the integration.

**Reply by ygaudet1967** [reply]
Yes, I've tried this and the Confluence Integration is there and I have enabled it, but there is no settings tab.

**Reply by andymac4182** [reply]
It should be on the left something like below once enabled in the normal `Community Plugins` section of settings.

<img width="236" alt="image" src="https://user-images.githubusercontent.com/1297073/235184980-be48517f-8239-49e6-af6b-1f4591fcc395.png">

**Reply by ygaudet1967** [reply]
This is what I see:

![image](https://user-images.githubusercontent.com/72083563/235185687-a20e07c7-b515-470e-97ab-e3eab9ef1c74.png)

**Reply by andymac4182** [reply]
Hi,

Thanks :) I missed an issue with some new settings validation when first installed. I missed due to having all my settings already available. I have a fix that will be ready shortly.

Cheers,
Andrew

**Reply by andymac4182** [reply]
I have just tested https://github.com/markdown-confluence/obsidian-integration/releases/tag/4.6.4 and it has the settings tab now appearing.

**Reply by ygaudet1967** [reply]
Looks good! Hoping I'll have time to try it out this afternoon.  Thanks.

![image](https://user-images.githubusercontent.com/72083563/235196318-5e181f80-df7d-4f9c-80d5-42932fda72ce.png)

**Comment by jokdis27** [comment]
My use case is to publish selected pages from my Obsidian Vault to Confluence. These pages-to-be-published are scattered here and there throughout my vault. I'm okay with having a flat page structure on Confluence, say, under a single root page called obsidian-vault or some such thing. 

So, basically, I want to be able to include the front matter
```
---
connie-publish: true
---
```
to any page, anywhere in my vault, and have such pages published to Confluence under the root page I configured in obsidian-confluence. Is this doable? For a single page I publish, it seems to work well, but for more than page, it seems to break.

**Reply by andymac4182** [reply]
So you want to turn off the folder structure feature and make all pages just be parented to the page defined in settings?

**Reply by jokdis27** [reply]
Yes. I can give you reasons, but that's the basics of it.

**Reply by jokdis27** [reply]
Even better (I think, ahem) would be to enable separate folder structures on Confluence and in Obsidian. But this seems much more difficult to me, and is (again, for me) only incrementally more valuable that the original ask.

**Reply by andymac4182** [reply]
Thanks. This has been a previous ask as well. I will have a look at this soon.

**Comment by silverjam** [comment]
Thanks for creating this plugin, I love writing in Obsidian so having an easy way to publish to Confluence is great.  Is there a plan to support labels in Confluence, or is this already a supported feature?

**Reply by andymac4182** [reply]
Hey,

Yes it is :) It translates tags in the YAML frontmatter (Properties in Obsidian) to labels https://obsidian-confluence.com/features/yaml-frontmatter.html#tags

👍

**Reply by silverjam** [reply]
:tada:

**Comment by song4onion** [comment]
I installed it with confluence-integration and set it up well.
![image](https://github.com/markdown-confluence/markdown-confluence/assets/147456962/a00a2cc2-ecf6-4dc2-b55e-b1593c0cc56d)
However, if you use the Ribbon Icon to publish, the
![image](https://github.com/markdown-confluence/markdown-confluence/assets/147456962/532a8e70-4bb7-4f1d-89bf-15a16f90c8fe)
![image](https://github.com/markdown-confluence/markdown-confluence/assets/147456962/8d64a59d-1563-4def-aa32-52def447293b)
Nothing is uploaded.
